### ISSUE TITLE
document.createElement does not create SVGGraphicsElement

### ISSUE THREAD

**Issue by runarberg** [capricorn86/happy-dom#1079]
**Describe the bug**

When you create an element with `document.createElement` or `document.createElementNS` it will incorrectly create some `SVGGraphicsElements` as `SVGElements`

**To Reproduce**

```js
const rect = document.createElementNS("http://www.w3.org/2000/svg", "rect");

rect.constructor.name
// SVGElement

rect instanceof SVGGraphicsElement
// false
```

**Expected behavior**

```js
const rect = document.createElementNS("http://www.w3.org/2000/svg", "rect");

rect.constructor.name
// SVGRectElement

rect instanceof SVGGraphicsElement
// true
```

**Comment by knownasilya** [comment]
Also this:

```
const element = document.createElement('input');

console.log(element instanceof HTMLBodyElement);
// ^ true
```

**Comment by onx2** [comment]
This seems to be a problem in general. I tested a few different elements and it is giving me the same result.

```ts
const element = document.createElement("span");
console.log(element instanceof HTMLBodyElement); // true
console.log(element instanceof HTMLDivElement); // true
```

```ts
const element = document.createElement("input");
console.log(element instanceof HTMLBodyElement); // true
console.log(element instanceof HTMLDivElement); // true
console.log(element instanceof HTMLSpanElement); // true
```

```ts
const element = document.createElement("div");
console.log(element instanceof HTMLBodyElement); // true
console.log(element instanceof HTMLSpanElement); // true
```

**Comment by onx2** [comment]
I tested each unimplemented one and they all were `true` for `span`, `div`, `input`, `ol`, `ul`, `li`, and I'm sure others as well:


```ts
console.log(element instanceof HTMLHeadElement); // true
console.log(element instanceof HTMLTitleElement); // true
console.log(element instanceof HTMLBodyElement); // true
console.log(element instanceof HTMLHeadingElement); // true
console.log(element instanceof HTMLParagraphElement); // true
console.log(element instanceof HTMLHRElement); // true
console.log(element instanceof HTMLPreElement); // true
console.log(element instanceof HTMLUListElement); // true
console.log(element instanceof HTMLOListElement); // true
console.log(element instanceof HTMLMenuElement); // true
console.log(element instanceof HTMLDListElement); // true
console.log(element instanceof HTMLDivElement); // true
console.log(element instanceof HTMLAreaElement); // true
console.log(element instanceof HTMLBRElement); // true
console.log(element instanceof HTMLCanvasElement); // true
console.log(element instanceof HTMLDataElement); // true
console.log(element instanceof HTMLDataListElement); // true
console.log(element instanceof HTMLDetailsElement); // true
console.log(element instanceof HTMLDirectoryElement); // true
console.log(element instanceof HTMLFieldSetElement); // true
console.log(element instanceof HTMLFontElement); // true
console.log(element instanceof HTMLHtmlElement); // true
console.log(element instanceof HTMLLegendElement); // true
console.log(element instanceof HTMLMapElement); // true
console.log(element instanceof HTMLMarqueeElement); // true
console.log(element instanceof HTMLMeterElement); // true
console.log(element instanceof HTMLModElement); // true
console.log(element instanceof HTMLOutputElement); // true
console.log(element instanceof HTMLPictureElement); // true
console.log(element instanceof HTMLProgressElement); // true
console.log(element instanceof HTMLQuoteElement); // true
console.log(element instanceof HTMLSourceElement); // true
console.log(element instanceof HTMLSpanElement); // true
console.log(element instanceof HTMLTableCaptionElement); // true
console.log(element instanceof HTMLTableCellElement); // true
console.log(element instanceof HTMLTableColElement); // true
console.log(element instanceof HTMLTableElement); // true
console.log(element instanceof HTMLTableRowElement); // true
console.log(element instanceof HTMLTableSectionElement); // true
console.log(element instanceof HTMLFrameElement); // true
console.log(element instanceof HTMLFrameSetElement); // true
console.log(element instanceof HTMLEmbedElement); // true
console.log(element instanceof HTMLObjectElement); // true
console.log(element instanceof HTMLParamElement); // true
console.log(element instanceof HTMLTrackElement); // true
```

**Comment by rijenkii** [comment]
I guess this is in the same boat:
```typescript
const g = document.createElementNS("http://www.w3.org/2000/svg", "g");
const check = g instanceof SVGGElement;
// ReferenceError: SVGGElement is not defined
```

**Comment by capricorn86** [comment]
Thank you for reporting @runarberg, @knownasilya, @onx2 and @rijenkii! :slightly_smiling_face: 

@onx2 This is because several HTML elements where not implemented yet. The remaining HTML elements where implemented in [v15](https://github.com/capricorn86/happy-dom/releases/tag/v15.0.0). As they where not implemented yet, they where assigned as HTMLElement.

All remaining SVG elements has now also been implemented in [v15.8.0](https://github.com/capricorn86/happy-dom/releases/tag/v15.8.0).
